bedrock/firefox/templates/firefox/sync.html

```diff
@@ -94,9 +94,30 @@
         </div>
 
         <div class="show-fx-31-signed-out">
+        {% if not version %}
           <button class="sync-button" data-interaction="button click" data-element-action="Sync CTA"  id="cta-sync">
             {{ _('Get started with Sync') }}
           </button>
+        {% else %}
+          <form id="fxa-email-form" action="{{ url('firefox.accounts') }}" method="get">
+            <input type="hidden" id="variation" name="v" value="{{ version }}">
```

Comment:
Yeah, coordinating tests is still a manual process. At least it's pretty clear in Optimizely which tests are currently running. :confused: